gateway-service-api/src/main/proto/org/hypertrace/gateway/service/v1/log_event.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "org.hypertrace.gateway.service.v1.log.events";
+
+package org.hypertrace.gateway.service.v1.log.event;
+
+import "org/hypertrace/gateway/service/v1/gateway_query.proto";
+
+message LogEventRequest {
```

Comment:
can we add the limit, order, offset etc. params

Reply:
Currently we want to support query on spanId/traceId, do we need limit and offset for it? also `order` isn't needed at the moment I guess.

Reply:
Added